src/uu/runcon/src/runcon.rs

```diff
@@ -0,0 +1,422 @@
+                .and_then(set_next_exec_context)
+                .and_then(|_| -> Result<()> { Err(execute_command(command, &options.arguments)) })
+            {
+                show_error!("{}.", report_full_error(&r));
+                return libc::EXIT_FAILURE;
+            }
+        }
+
+        CommandLineMode::CustomContext {
+            compute_transition_context,
+            user,
+            role,
+            the_type,
+            range,
+            command,
+        } => {
+            if let Some(command) = command {
+                if let Err(r) = get_custom_context(
+                    *compute_transition_context,
+                    user.as_deref(),
+                    role.as_deref(),
+                    the_type.as_deref(),
+                    range.as_deref(),
+                    command,
+                )
+                .and_then(set_next_exec_context)
+                .and_then(|_| -> Result<()> { Err(execute_command(command, &options.arguments)) })
+                {
+                    show_error!("{}.", report_full_error(&r));
+                    return libc::EXIT_FAILURE;
+                }
+            } else if let Err(r) = print_current_context() {
+                show_error!("{}.", report_full_error(&r));
+                return libc::EXIT_FAILURE;
+            }
+        }
+    }
+
+    libc::EXIT_SUCCESS
+}
+
+pub fn uu_app() -> App<'static, 'static> {
+    App::new(executable!())
+        .version(VERSION)
+        .about(ABOUT)
+        .after_help(DESCRIPTION)
+        .arg(
+            Arg::with_name(options::COMPUTE)
+                .short("c")
+                .long(options::COMPUTE)
+                .takes_value(false)
+                .help("Compute process transition context before modifying."),
+        )
+        .arg(
+            Arg::with_name(options::USER)
+                .short("u")
+                .long(options::USER)
+                .takes_value(true)
+                .value_name("USER")
+                .help("Set user USER in the target security context."),
+        )
+        .arg(
+            Arg::with_name(options::ROLE)
+                .short("r")
+                .long(options::ROLE)
+                .takes_value(true)
+                .value_name("ROLE")
+                .help("Set role ROLE in the target security context."),
+        )
+        .arg(
+            Arg::with_name(options::TYPE)
+                .short("t")
+                .long(options::TYPE)
+                .takes_value(true)
+                .value_name("TYPE")
+                .help("Set type TYPE in the target security context."),
+        )
+        .arg(
+            Arg::with_name(options::RANGE)
+                .short("l")
+                .long(options::RANGE)
+                .takes_value(true)
+                .value_name("RANGE")
+                .help("Set range RANGE in the target security context."),
+        )
+        .arg(Arg::with_name("ARG").multiple(true))
```

Comment:
You'll need to set the `AppSettings::TrailingVarArg` setting for the app so that everything after the command is treated as  an argument for the command. `runcon true -j 2` for example does not work right now

Reply:
I was looking for this option, but I couldn't find how to do this with `clap`. Thank you for the hint!